content/usage/advanced/index.md

```diff
@@ -237,44 +445,90 @@ There are buttons to
     - this may move around if the control station computer is on a boat
 1. move the map to follow the vehicle's current position
 1. download the current mission from the vehicle, and display it on the map
-1. execute the mission that is on the vehicle
+1. run the mission that is on the vehicle
 {{ easy_image(src="map-widget", width=350, center=true) }}
-{{ easy_image(src="map-config", width=350, center=true) }}
 
-In future it will be possible to set the current vehicle position, and click to guide the vehicle to new
```

Comment:
Nice to mention also that the user can toggle between "structured" map and "satellite" map, and that there are markers for sea levels and objects that can be toggled.